docs/fiddles/features/drag-and-drop/main.js

```diff
@@ -1,20 +1,23 @@
 const { app, BrowserWindow, ipcMain, nativeImage, NativeImage } = require('electron')
-const fs = require('fs');
-const http = require('http');
+const path = require('path')
+const fs = require('fs')
+const http = require('http')
 
 function createWindow () {
   const win = new BrowserWindow({
     width: 800,
     height: 600,
     webPreferences: {
-      nodeIntegration: true
+      preload: path.join(__dirname, 'preload.js')
     }
   })
 
   win.loadFile('index.html')
 }
-const iconName = 'iconForDragAndDrop.png';
-const icon = fs.createWriteStream(`${process.cwd()}/${iconName}`);
+
+const iconName = `${process.cwd()}/iconForDragAndDrop.png`;
```

Comment:
What do you think about using `path.join` the same way as with preload for consistency with `preload` and avoiding possible issues with Windows?

```suggestion
const iconName = path.join(process.cwd(), 'iconForDragAndDrop.png');
```